### DISCUSSION TITLE
.Setup.X().Runs

### DISCUSSION THREAD

**Discussion by jsk95** [ashmind/SourceMock#7] [Ideas]
Might need something like `.Runs` as minimum to provide some flexibility + before `.Throws` is available.

**Comment by jsk95** [comment]
Or maybe `Returns` could just take in a lambda

**Reply by ashmind** [reply]
That is kind of planned, but the types are complicated enough to postpone (not _a_ lambda, lambda params must match original method args).

**Comment by ashmind** [comment]
`.Throws` is available :)
